examples/image_classification.ipynb

```diff
@@ -0,0 +1,5278 @@
+{
```

Comment:
<p>remove this if you do changes I showed above in installation step</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/143/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>